tensorboard/plugins/core/core_plugin_test.py

```diff
@@ -172,7 +172,10 @@ def testIndex_returnsActualHtml(self):
         self.assertEqual(200, response.status_code)
         self.assertStartsWith(response.headers.get("Content-Type"), "text/html")
         html = response.get_data()
-        self.assertEqual(html, FAKE_INDEX_HTML)
+        self.assertEqual(
+            html,
+            b'<meta name="tb-relative-root" content="/" />' + FAKE_INDEX_HTML,
```

Comment:
Can we add a couple more tests for the path prefix behavior?  We could add a new TestCase that doesn't construct the WSGI app in `setUp`, and instead have each test method call `TensorBoardWSGI` with a different value of the `path_prefix` parameter - maybe e.g. empty, one segment, and two segments.

It would also be good to try at least a couple different values for the path we pass to `get()` - e.g. "/", "/index.html", "/experiment/123/", "/experiment/123" (no trailing slash), etc.